### DISCUSSION TITLE
2 questions on "form"

### DISCUSSION THREAD

**Discussion by extralam** [filamentphp/filament#2220] [Help]
1. is there something like a "modal" concept in "form"?
Use case: Like Creating a quotation for new customers, the select box does not include this customer. I want to quickly create this new customer directly with a button and popup a dialog to create.

2. My use case is 
"Invoice" Has Many "Invoice Item" 
"Invoice Item" Belongs To "Invoice Template"
I want to edit "Invoice item" in the "Invoice" form, but some config is getting from "Invoice Template"

**Comment by ryangjchandler** [comment]
@extralam In response to number 1, #2057 is a work in progress and will hopefully be ready soon.
